### DISCUSSION TITLE
No screen transitions working on android 😥.

### DISCUSSION THREAD

**Discussion by siddharthkr** [software-mansion/react-native-screens#1402] [Q&A] [answered]
I am using react-navigation v6 and none of the transition is working https://reactnavigation.org/docs/stack-navigator/#animations

**Comment by nrwinner** [comment, marked as answer]
It looks like the docs you linked are not for the Native Stack Navigator. The regular Stack Navigator is maintained by the React Navigation team so you might consider moving your question there 🙂 Additionally it would be helpful to have a limited reproduction of your issue with the question. It's very hard for us to debug if we can't see your code.

**Comment by siddharthkr** [comment]
@nrwinner Thanks, actually for some reason, it started working now (I might be doing something wrong). 🙂
